diff --git a/dist/vanilla-extract-css.cjs.d.ts b/dist/vanilla-extract-css.cjs.d.ts
index b5412c187b66dc3ebe93c32e2cc4e9aff043fa48..89815291f8f07698dd63f094f352d83c44e1e31b 100644
--- a/dist/vanilla-extract-css.cjs.d.ts
+++ b/dist/vanilla-extract-css.cjs.d.ts
@@ -1,11 +1,11 @@
 import { Properties, AtRule } from 'csstype';
 
-type CSSVarFunction = `var(--${string})` | `var(--${string}, ${string | number})`;
-type Contract = {
+export type CSSVarFunction = `var(--${string})` | `var(--${string}, ${string | number})`;
+export type Contract = {
     [key: string]: CSSVarFunction | null | Contract;
 };
 type Primitive = string | boolean | number | null | undefined;
-type MapLeafNodes<Obj, LeafType> = {
+export type MapLeafNodes<Obj, LeafType> = {
     [Prop in keyof Obj]: Obj[Prop] extends Primitive ? LeafType : Obj[Prop] extends Record<string | number, any> ? MapLeafNodes<Obj[Prop], LeafType> : never;
 };
 
